### PR TITLE
Switch to github actions

### PR DESCRIPTION
**Pull request by ale-rt**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('setup.*', '*.cfg') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install -r requirements.txt
+          sudo apt-get install libxml2-dev libxslt-dev xvfb
```
Comment (mauritsvanrees) [inline .github/workflows/tests.yml:35]: Why are these apt dependencies needed?
I can imagine the first two, for compiling lxml, but then I wonder why they are not needed by other Plone projects. Or are they?
`xvfb` should not be needed: we do not need an X server for running a browser.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      - name: Run buildout
+        run: buildout -c .gha.cfg
```
Comment (mauritsvanrees) [inline .github/workflows/tests.yml:37]: I would leave out the dot: I think I prefer the file to be visible for clarity.

Comment (mauritsvanrees) [inline .github/workflows/tests.yml:37]: I would leave out the dot: I think I prefer the file to be visible for clarity.

**Review by mauritsvanrees — CHANGES_REQUESTED** [review]
Thanks!
Mostly good, a few minor comments inline.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (ale-rt) [reply .github/workflows/tests.yml:35]: I saw the build failing like this:
```
ERROR: b'/bin/sh: 1: xslt-config: not found\n'
** make sure the development packages of libxml2 and libxslt are installed **
```
See https://github.com/collective/collective.recipe.backup/runs/2585100630?check_suite_focus=true
Then I copied-pasted that line from another GHA actions wf.
Obviously `xvfb` is not needed. I will remove it.

**Review by mauritsvanrees — APPROVED** [review]

**Comment by mauritsvanrees** [comment]
Thanks a lot!

File: .github/workflows/tests.yml (hunk 1 above)
Comment (mauritsvanrees) [reply .github/workflows/tests.yml]: I see that in our [buildout config](https://github.com/collective/collective.recipe.backup/blob/master/buildout.cfg) we pin lxml 4.3.3, which has wheels, but we use a somewhat old `zc.buildout` version. Well, 2.13.1 should be able to handle wheels. Anyway, this buildout config can use some love.
We might not need it at all. Well, making sure this recipe runs in an actual buildout is good. Anyway, something for another day.

**Pull request merged**